.github/CONTRIBUTING.md

```diff
@@ -17,7 +17,7 @@ For more detailed information on contribution please read our [beginners guide](
 
 1. Contributions must adhere to the [Magento coding standards](https://devdocs.magento.com/guides/v2.4/coding-standards/bk-coding-standards.html).
 2. Pull requests (PRs) must be accompanied by a meaningful description of their purpose. Comprehensive descriptions increase the chances of a pull request being merged quickly and without additional clarification requests.
-3. Commits must be accompanied by meaningful commit messages. Please see the [Magento Pull Request Template](https://github.com/magento/magento2/blob/2.3-develop/.github/PULL_REQUEST_TEMPLATE.md) for more information.
+3. Commits must be accompanied by meaningful commit messages. Please see the [Magento Pull Request Template](https://github.com/magento/magento2/blob/2.4-develop/.github/PULL_REQUEST_TEMPLATE.md) for more information.
```

Comment:
Please use a stable URL as suggested in https://stackoverflow.com/a/65158264.

After changes are applied and all builds are green, please squash them into a single commit so that we have perfectly clean history 😉

Reply:
Sure @orlangur, Thanks!

Reply:
@orlangur I have updated the commit, Please proceed.